unit_testing/test_labels.py

```diff
@@ -139,11 +139,31 @@ def test_compute_mean_squared_error():
     for x, y, z, _ in src.getNonZeroCoordinates():
         if z < 5:
             ref.data[x, y, z+2] = src.data[x, y, z]
-
     mse = sct_labels.compute_mean_squared_error(src, ref)
     assert mse == 1.1547005383792515
 
 
+def test_compute_mse_label_warning(caplog):
+    src = fake_3dimage_sct()
+    ref = src.copy()
+
+    src.data = np.where(src.data == ref.data[0, 0, 0], 1500, ref.data)
```

Comment:
It might be good to add the same comment you wrote for the other test ("modifying the data to create label mismatch", or "false positive/false negative") just to clarify what this operation does.

Reply:
Done